scripts/__tests__/build-blog-data-sanitization.test.js

```
@@ -0,0 +1,475 @@
+      const malformedHtml =
+        '<div><p>Unclosed paragraph<div>Nested incorrectly</div>'
+      const sanitized = sanitizeHtml(malformedHtml, sanitizeOptions)
+
+      // Should not throw and should contain the text content
+      expect(sanitized).toContain('Unclosed paragraph')
+      expect(sanitized).toContain('Nested incorrectly')
+    })
+
+    test('should handle deeply nested content', () => {
+      const deeplyNested =
+        '<div>'.repeat(100) + 'Content' + '</div>'.repeat(100)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)